tools/offcputime.py

```diff
@@ -177,7 +182,15 @@ def signal_ignore(signal, frame):
 else:
     thread_context = "all threads"
     thread_filter = '1'
+if args.state == 0:
+    state_filter = 'prev->state == 0'
+elif args.state != 999:
```

Comment:
We don't need a special value here — if we drop the default `state` will either be a positive integer or be `None`. This code can then be:

```python
if args.state == 0:
    state_filter = 'prev->state == 0'
elif args.state:
    # these states are sometimes bitmask checked
    state_filter = 'prev->state & %d' % args.state
else:
    state_filter = '1'
```